assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -151,6 +159,20 @@ const OrderSummaryItem = ( { cartItem } ) => {
 					variation={ variation }
 				/>
 			</div>
+			<span className="screen-reader-text">
+				{ sprintf(
+					/* translators: %1$d is the number of items, %2$s is the total price including the currency symbol and %3$s is the item name. */
+					_n(
+						'Total price for %1$d %3$s item: %2$s',
+						'Total price for %1$d %3$s items: %2$s',
+						quantity,
+						'woo-gutenberg-products-block'
+					),
+					quantity,
+					formatPrice( subtotalPrice, totalsCurrency ),
+					name
+				) }
+			</span>
 			<div className="wc-block-components-order-summary-item__total-price">
```

Comment:
I would add `aria-hidden="true"` here, otherwise the price is read twice by screen readers.
```suggestion
			<div className="wc-block-components-order-summary-item__total-price" aria-hidden="true">
```

Reply:
Added now!